### PR TITLE
HPCC-16009 Improve the code for CRowStreamReader with no limit

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@jakesmith 
I'm not sure if this is worth it.  Timings didn't show any significant benefit, but the generated code is slightly better.  See what you think/.

**Comment by ghalliday** [comment]
Jira: https://track.hpccsystems.com/browse/HPCC-16009

File: common/thorhelper/thorcommon.cpp [hunk 1]
```diff
@@ -1156,19 +1155,17 @@ class CRowStreamReader : public CSimpleInterface, implements IExtRowStream
...
         : fileio(_fileio), mmfile(_mmfile), allocator(rowif->queryRowAllocator()), prefetchBuffer(NULL) 
     {
 #ifdef TRACE_CREATE
         PROGLOG("CRowStreamReader %d = %p",++rdnum,this);
 #endif
-        maxrows = _maxrows;
         grouped = _grouped;
         eoi = false;
-        eos = maxrows==0;
+        eos = false;
```
Comment (jakesmith) [inline common/thorhelper/thorcommon.cpp]: style question: I think we favour initializers at member declaration level now?

**Comment by jakesmith** [comment]
Change looks clean and improvement.
Can't see any issues.

@ghalliday - ready to merge.

**Comment by ghalliday** [comment]
@richardkchapman

**Pull request merged**